test/functional/rpc_psbt.py

```diff
@@ -187,6 +188,16 @@ def run_test(self):
         # Create and fund a raw tx for sending 10 BTC
         psbtx1 = self.nodes[0].walletcreatefundedpsbt([], {self.nodes[2].getnewaddress():10})['psbt']
 
+        self.log.info("Test that max_tx_weight option is sanity checked and fails when expected")
+        dest_arg = [{self.nodes[0].getnewaddress(): 1}]
+        min_tx_weight = MIN_STANDARD_TX_NONWITNESS_SIZE * WITNESS_SCALE_FACTOR
+        assert_raises_rpc_error(-8, f"Invalid parameter, max tx weight must be between {min_tx_weight} and 400000", self.nodes[0].walletcreatefundedpsbt, [], dest_arg, 0, {"max_tx_weight": -1})
```

Comment:
I can see that the "400000" here is the `MAX_STANDARD_TX_WEIGHT` but I think it's better to define the `MAX_STANDARD_TX_WEIGHT` constant and use it here. This makes it obvious to anyone reading what "400000" is.

Reply:
Yes can also be used in future tests, thinking about defining it in 
`test_framework/blocktools.py`

Reply:
Fixed thanks